moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -323,26 +337,35 @@ local function remove_objects(msg, doc, section, objects)
 end
 
 
+local function validate_schema(hsr, msg, doc, version)
```

Comment:
There are several way this can be accomplished but until the schema deployment mechanism is defined it would be premature.   Currently when new schemas are deployed the plugins are restarted meaning they are automatically re-loaded.